src/applications/personalization/profile360/components/ContactInformation.jsx

```diff
@@ -21,6 +21,10 @@ import Email from '../vet360/components/Email';
 import { handleDowntimeForSection } from './DowntimeBanner';
 import ContactInformationExplanation from './ContactInformationExplanation';
 
+import isBrandConsolidationEnabled from '../../../../platform/brand-consolidation/feature-flag';
+
+const propertyName = isBrandConsolidationEnabled() ? 'VA.gov' : 'Vets.gov';
```

Comment:
"property" makes sense to use here, but is a little unclear at first glance (unless this becomes the standard term for distinguishing these), could "host" be clearer without being misleading?

Reply:
I thought about that but wanted to differentiate it from something that has a technical implication- this is only used for content that references the site as an brand / entity. (e.g. it needs to be capitalized and should omit the protocol)